src/twisted/python/_dist3.py

```diff
@@ -0,0 +1,185 @@
+# -*- test-case-name: twisted.python.test.test_dist3 -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+Python 3 porting metadata. Only necessary while parts of Twisted are unported.
+
+@var notPortedModules: A list of fully-qualified Python names that correspond
+    to modules in Twisted are not ported, for exclusion during installation.
+"""
+
+from __future__ import absolute_import, division
+
+notPortedModules = [
+    "twisted.conch.tap",
+    "twisted.conch.ttymodes",
+    "twisted.internet._ssl",
```

Comment:
There is: https://github.com/twisted/twisted/pull/501/files/9535a4069979e35226c6a3a329bd154f1c586604#diff-8df27befe87e3feb795ca53f050639a5L29 -- we removed this file a few hours ago and I haven't merged forward. :)